JSON/include/Poco/JSON/Object.h

```diff
@@ -64,12 +64,16 @@ class JSON_API Object
 	typedef ValueMap::value_type                ValueType;
 	typedef ValueMap::iterator                  Iterator;
 	typedef ValueMap::const_iterator            ConstIterator;
+	typedef std::vector<std::string>            NameList;
 
-	explicit Object(bool preserveInsertionOrder = false);
+	explicit Object(bool preserveInsertionOrder = false, bool escapeUnicode = false);
```

Comment:
Maybe we should think about introducing an options enum for the various places where we use preserveInsertionOrder and escapeUnicode, and use an `int options` instead of a potentially growing list of bool args. If we just change the Object constructor to:

```
Object(int options = 0);
```

and define:
```
enum Options
{
    JSON_OPT_PRESERVE_INSERTION_ORDER = 1,
    JSON_OPT_ESCAPE_UNICODE = 2
};
```
Then the new constructor taking `int` will be backwards compatible to the one taking `bool`.